src/database/prisma/schema.prisma

```diff
@@ -571,6 +574,8 @@ model Payment {
     @@map("payment")
 }
 
+/// @@allow('read', auth().memberId == fromMemberId || auth().memberId == toMemberId)
+/// @@allow('update', has(auth().policies, 'core:member:ping'))
```

Comment:
Should this include the above check as well, ie that the user is one of the included members in the ping?